cmd/clair-load-test/reports.go

```diff
@@ -203,32 +236,46 @@ func orchestrateWorkload(ctx context.Context, manifests [][]byte, manifestHashes
 		"ESIndex":     conf.ESIndex,
 		"Host":        conf.Host,
 	}
+	time.Sleep(30 * time.Second)
```

Comment:
I don't understand why this is here -- it's a real code smell to unconditionally sleep for no reason.

Reply:
Sorry about that. Was doing some testing and forgot to remove it.